src/components/ListOptions/styles.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../theme/variables.scss';
```

Comment:
looks like we don't need any vars?

Reply:
I was told not to worry about styling this (UX guys want to control this sort of thing).  I went above and beyond to a degree...  Made a scaffold so the styling can be done later by editing the existing files.

Reply:
uses $red now, as you asked